wordpress-mobile/WordPress-Android
# Issue/13326 here to help

Parent #13326 

This PR adds support for clickable "here to help" text in the scan state having threats.


https://user-images.githubusercontent.com/1405144/106588333-a988f600-6570-11eb-81c0-d8746ace85d6.mov

To test

Prerequisites:

- A threat is added to the testing site. In case you run out of threats, re-install bad calendar plugin and re-run scan.
- Scan feature flag is enabled

1. Open Scan screen
2. Start scan
3. Notice that description on the top includes a clickable "here to help" text
4. Click on "here to help" 
5. Notice that help screen is shown

Merge Instructions:

1. Make sure PR #13940 is merged to develop
2. Remove "Not Ready for Merge" label
3. Merge this PR

PR submission checklist:
